docs/userguide/containers/dockerrepos.md

```diff
@@ -42,6 +42,7 @@ You can now commit and push your own images up to your repos on Docker Hub.
 > **Note:**
 > Your authentication credentials will be stored in the `~/.docker/config.json`
 > authentication file in your home directory.
+> For Windows users, this Docker file is stored in `$HOME/.docker/config.json`
```

Comment:
Being a Linux user I prefer the `~/` notation anytime but I don't mind updating.
In any case, the addition of Windows/OSx users would be good IMO, it's not a lot of extra text.